pomcorn/locators/__init__.py

```diff
@@ -11,3 +16,20 @@
     TagNameLocator,
     TextAreaByLabelLocator,
 )
+
+__all__ = [
+    "Locator",
```

Comment:
Currently ordering in `__all__` section differs from ordering in "import" section.

Let's either put them in the same order, or use some plugin to verify this, example - https://github.com/tlocke/flake8-alphabetize

Reply:
`flake8-alphabetize`, unfortunately, doesn't work for us, because
1. It has different sorting rules with `isort`
2. We ignore `__init__.py` files in our flake8 config

I found that isort has an [option](https://github.com/PyCQA/isort/blob/main/docs/configuration/options.md#sort-re-exports) to sort the `__all__` section, but now it is not working correctly: https://github.com/PyCQA/isort/issues/2193#issuecomment-1811756794

So I suggest to organize `__all__` section manually for now, and when isort fixes this bug, we will add this option to config.